pkg/nsx/util/errors.go

```diff
@@ -607,14 +608,15 @@ var (
 
 type RealizeStateError struct {
 	message string
+	Code    int
```

Comment:
`Code` is also not for external access, as the same as message, it it right?

Reply:
I missed that you have used it in subnetport to check error code, so, it should be ok.